lib/routes/account.js

```diff
@@ -646,6 +734,7 @@ module.exports = function (
             response.verificationMethod = 'email'
             response.verificationReason = 'login'
           }
+          recordSecurityEvent() // don't block response recording db event
```

Comment:
Are you referring to this section of [createResponse](https://github.com/mozilla/fxa-auth-server/blob/fee4cd9f752092563137bac72a81cb639a269778/lib/routes/account.js#L730)? I think it is ok to log a security event even if they are not requesting keys?

To clear up some of your questions, all sessions are created unverified, however we had to add an additional property [mustVerify](https://github.com/mozilla/fxa-auth-server/blob/fee4cd9f752092563137bac72a81cb639a269778/lib/routes/account.js#L730) to give us a better indication of what the session was being used for and if it even needed to be verified. Sessions can only be verified through a sign-in email, and if they were not requesting keys, they would not get the mail, therefore they currently can  never verify that session. 

We could eventually add the ability verify a session other means though.